src/main/java/jenkins/scm/api/mixin/ChangeRequestSCMHead.java

```diff
@@ -56,4 +80,11 @@
     @NonNull
     SCMHead getTarget();
 
+    /**
+     * Returns the title of the change request
+     * @return title of the change request
+     */
+    default String getTitle() {
+        return "";
```

Comment:
One suggestion for default might be something like "Merge {target} into {base}

Reply:
If a user gets a title which is not the real title. It might be wired. How about letting it returns a Null? Plus, add an annotation `CheckForNull` to it.

Reply:
@LinuxSuRen 
I'm not sure about this method being added. 
I do not think the title of the ChangeRequest should be part of the SCM information.  
What use cases do you see for this?   
Is there a JIRA for this?